pinot-spi/src/main/java/org/apache/pinot/spi/config/table/ingestion/BatchIngestionConfig.java

```diff
@@ -35,31 +35,31 @@
   @JsonPropertyDescription("Configs for all the batch sources to ingest from")
   private final List<Map<String, String>> _batchConfigMaps;
 
-  @JsonPropertyDescription("Push type APPEND or REFRESH")
-  private final String _segmentPushType;
+  @JsonPropertyDescription("Load type APPEND or REFRESH")
+  private final String _segmentLoadType;
 
-  @JsonPropertyDescription("Push frequency HOURLY or DAILY")
-  private final String _segmentPushFrequency;
+  @JsonPropertyDescription("Load frequency HOURLY or DAILY")
+  private final String _segmentLoadFrequency;
 
   @JsonCreator
   public BatchIngestionConfig(@JsonProperty("batchConfigMaps") @Nullable List<Map<String, String>> batchConfigMaps,
-      @JsonProperty("segmentPushType") String segmentPushType,
-      @JsonProperty("segmentPushFrequency") String segmentPushFrequency) {
+      @JsonProperty("segmentLoadType") String segmentLoadType,
+      @JsonProperty("segmentLoadFrequency") String segmentLoadFrequency) {
```

Comment:
Alternative suggest" `segmentIngestionFrequency` and `segmentIngestionType`, since "load" has the connotation of a segment being loaded in memory. Especially with the recent proposal for a segment to be in "deep store" vs "loaded"

Reply:
make sense, i will make the change